pkg/tfvars/vsphere/vsphere.go

```diff
@@ -52,14 +53,6 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 	// /<datacenter>/vm/<folder_path> so we can split on "vm/".
 	folderRelPath := strings.SplitAfterN(controlPlaneConfig.Workspace.Folder, "vm/", 2)[1]
 
-	// The vSphere provider needs explicit path of the datacenter to avoid
-	// conflicts when multiple datacenters have the same name even though they
-	// are in different folders.
-	datacenter := controlPlaneConfig.Workspace.Datacenter
-	if !strings.HasPrefix(datacenter, "/") && !strings.HasPrefix(datacenter, "./") {
-		datacenter = "./" + datacenter
```

Comment:
The old code was prepending "./". The new code is prepending "/". Also, the old code was accepting "./" as a fully-qualifying prefix. Is there a difference between "/" and "./"?

Reply:
The `./` was work that I did for the routers that came over (copy and paste) to datacenters. The datacenters should not be using `./` in their path as there is no reference for "current working directory" for this type of object. As a result, the it would have become the root path anyhow. I used `./` in the prior PR[1] to force a qualified path not realizing this. However, moving to fully qualified paths achieves the same goal with additional benefits. Keeping in mind, both of these PRs should land in the same release, this is operationally the same at this place in the code.

[1] https://github.com/openshift/installer/pull/4894

Reply:
Sounds good.

For my clarity, is it a bad datacenter name if the user specifies a datacenter name that starts with "./"?

Reply:
If they specify a name starting with "./", it will end up being "/./", which technically works and will operationally do the same thing as before this change. This is what was happening with the changes in #4894.

Reply:
Thanks for the explanation.